playbooks/technical-planning.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Technology choices
+description: How to write technical plans at Artsy
+---
+
+# Technical planning
+
+Technical planning is a crucial part of our engineering process. Writing tech plans helps us:
```

Comment:
In my case, writing tech plans have also been extremely useful to understand how things currently work. Those things are usually spread across old PRs, slack threads, or in people's minds.